code/game/machinery/clonescanner.dm

```diff
@@ -197,3 +197,22 @@
 
 /obj/machinery/clonescanner/force_eject_occupant(mob/target)
 	remove_mob()
+
+/obj/machinery/clonescanner/crowbar_act(mob/user, obj/item/I)
+	if(!panel_open)
+		return
+	. = TRUE
+	if(!I.use_tool(src, user, 0, volume = I.tool_volume))
+		return
+	if(flags & NODECONSTRUCT)//We need to check for this early or the contents could be moved before it checks for the flag normally
+		return
+	for(var/obj/thing in contents) // in case there is something in the scanner
```

Comment:
This will never find anything because you are looking for `obj` things. Look at the cloner's definition: `var/mob/living/carbon/human/occupant` The thing inside that you want to remove is a `mob`. 

(I think this currently will never be called, because in `screwdriver_act` you simply stop deconstruction if there is an occupant)

Reply:
Yeah this seems unnecessary tbh
IIRC the default deconstruction also takes care of it

Reply:
seems so, ive removed it